notebooks/patches_data_viz.py

```diff
@@ -0,0 +1,82 @@
+import marimo
+
+__generated_with = "0.13.15"
+app = marimo.App(width="medium")
+
+@app.cell
+def _():
+    import marimo as mo
+    import pandas as pd
+    import polars as pl
+    import os
+    import matplotlib.pyplot as plt
+    return mo, pd, pl, os, plt
+
```

Comment:
Consider adding a quick summary of what the notebook is, what data is reading and how that data was created .

(Would be good to have after code changes are done :D )